mappings/net/minecraft/PowderSnowBucketItem.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5634 net/minecraft/PowderSnowBucketItem
```

Comment:
Needs to be moved to nm.item

Reply:
Fixed in 9c5142df05762088a9a6608e81dc2df8c3518239